pkgs/build-support/fetchgitlab/default.nix

```diff
@@ -30,5 +30,5 @@ let
   }) // passthruAttrs // { inherit name; };
 in
 
-fetcher fetcherArgs // { meta.homepage = "${protocol}://${domain}/${slug}/"; inherit rev; }
```

Comment:
Seems like `fetchFromGitHub` also doesn't put these in `passthru`, and whether that is a good thing is debatable.
This change will affect the environ of the FOD fetcher.